examples/albert/run_trainer.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This makes it possible to run that as `./run_trainer.py` even on the Ubuntu versions where `python2` is the default.